src/store/assets/assets-state.ts

```diff
@@ -0,0 +1,13 @@
+import { AssetsInterface } from '../../interfaces/assets.interface';
+import { LoadableEntityState } from '../types';
+
+export interface AssetsState extends LoadableEntityState<AssetsInterface[]> {}
```

Comment:
I think you should move tokens under a separate key, as this state will also contain other information 
```
interface AssetsState {
    tokens: LoadableEntityState<AssetsInterface[]>;
    prices: LoadableEntityState<PriceInterface[]>;
    ...
}
````